.travis.yml

```diff
@@ -8,25 +8,23 @@ env:
     - APP_MAUTH_UUID=398ea708-50eb-499f-98d3-14cc7692668c
 cache:
   directories:
-    - $HOME/.sbt
-    - $HOME/.ivy2/cache
-    - $HOME/.cache
-    - $HOME/.m2
+    - "$HOME/.ivy2/cache"
+    - "$HOME/.sbt/boot/"
 before_cache:
   - find $HOME/.ivy2 -name "ivydata-*.properties" -delete
   - find $HOME/.sbt -name "*.lock" -delete
 before_install:
-  - export APP_MAUTH_PRIVATE_KEY=$(cat .travis/fake_private_key_for_testing)
+  - export APP_MAUTH_PRIVATE_KEY=$(cat travis/fake_private_key_for_testing)
   - echo -e "machine github.com\n  login $GITHUB_AUTH_TOKEN" >> ~/.netrc
-jobs:
-  include:
-    - stage: test
-      script: sbt scalafmtSbtCheck scalafmtCheckAll +test +mimaReportBinaryIssues
-    - stage: release
-      script: 
-        - .travis/deploy.sh
-        - sbt mauth-proxy/ecr:push
-      if: tag IS NOT present
-stages:
-  - test
-  - release
+script: sbt scalafmtSbtCheck scalafmtCheckAll test
```

Comment:
was mimaReportBinaryIssues removed on purpose?

Reply:
good spot, reverted the change